tests/acceptance/new-test.js

```diff
@@ -561,6 +583,30 @@ describe('Acceptance: ember new', function () {
       checkFileWithEmberCLIVersionReplacement(fixturePath, 'tests/dummy/config/ember-cli-update.json');
     });
 
+    if (isExperimentEnabled('PNPM')) {
+      it('addon + pnpm + welcome', async function () {
+        await ember(['addon', 'foo', '--skip-npm', '--skip-git', '--pnpm', '--welcome']);
+
+        let fixturePath = 'addon/pnpm';
+
+        [
+          'tests/dummy/config/ember-try.js',
+          'tests/dummy/app/templates/application.hbs',
+          '.github/workflows/ci.yml',
+          'README.md',
+          'CONTRIBUTING.md',
+          '.npmrc',
+        ].forEach((filePath) => {
+          checkFile(filePath, path.join(__dirname, '../fixtures', fixturePath, filePath));
+        });
+
+        expect(file('.travis.yml')).to.not.exist;
```

Comment:
Again seems weird. I wonder what the absence of .travis.yml was supposed to be testing.

Reply:
Since travis is [still in the blueprint](https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/.travis.yml), i would bet that the assertion is, at most, half of the checks you'd need for supporting multiple ci providers.

With any ci provider option, you'd need to
- assert what you want exists in the output
- assert that the other options don't exist in the output